src/Node/FS/Async.purs

```diff
@@ -75,8 +81,18 @@ foreign import fs ::
   , writeFile :: forall a opts. Fn4 FilePath a { | opts } (JSCallback Unit) Unit
   , appendFile :: forall a opts. Fn4 FilePath a { | opts } (JSCallback Unit) Unit
   , exists :: forall a. Fn2 FilePath (Boolean -> a) Unit
+  , open :: Fn3 FilePath String (JSCallback FileDescriptor) Unit
+  , read :: Fn6 FileDescriptor Buffer BufferOffset BufferLength FilePosition (JSCallback ByteCount) Unit
+  , write :: Fn6 FileDescriptor Buffer BufferOffset BufferLength FilePosition (JSCallback ByteCount) Unit
+  , close :: Fn2 FileDescriptor (JSCallback Unit) Unit
   }
 
+foreign import create :: Fn4 FilePath String FileMode (JSCallback FileDescriptor) Unit
```

Comment:
OK - I called it this to be consistent with the existing Sync.purs module:

https://github.com/purescript-node/purescript-node-fs/blob/master/src/Node/FS/Sync.purs#L95